### ISSUE TITLE
[Bug] New Game - Downloading Scenarios Forever

### ISSUE THREAD

**Issue by DanielNelHeck** [NPBruce/valkyrie#1675]
Hello!

New user here, just downloaded Valkyrie in my Samsung Tab A9+ running Android 14 (UI 6.0).
Installed base MoM app (straight from Play Store not Steam)
Installed Valkyrie and granted all permissions it requested.
Imported content (and reimported too) to same Valkyrie folder of the installation itself
Selected my expansions

When I click on New Game it keeps downloading scenarios forever (blue Downloading... in the top right corner), not listing a single one.
Wifi is good, rebooted, disconnected, reconnected, etc, but nothing makes it download and list the scenarios.

Thanks in advance!
Daniel

**Comment by vidimus** [comment]
Hello, @DanielNelHeck 
Sometimes the Valkyrie server is offline, showing the problem you have reported. It usually takes a variable amount of time to get back to normality (less than 1 hour in most cases).
I have tried the connection yesterday in several times and all looks fine. Could you retry it?

**Comment by DanielNelHeck** [comment]
I tried multiple times and nothing.
Tried using 4G hotspot and one guest network, none of them worked.

Tried using hotel's guest yesterday and it WORKED :)
But now I have another issue and I'll be opening a new topic for that:
Scenarios are listed, content is downoaded, I can play the scenario and hear the intro, as soon as I click in CONTINUE to begin setup the app CRASHES and send me to home screen. Tested with multiple scenarios in multiple languages.
